plugins/by-name/flutter-tools/settings-options.nix

```diff
@@ -3,81 +3,102 @@ let
   inherit (lib) types;
   inherit (lib.nixvim)
     defaultNullOpts
-    literalLua
     mkNullOrOption'
     mkNullOrOption
     ;
 in
 {
   ui = {
-    border = defaultNullOpts.mkNullable (with types; either str (listOf (maybeRaw str))) "rounded" ''
-      The border type to use for all floating windows, the same options/formats used for
-      `:h nvim_open_win` e.g. `"single"` | `"shadow"` | `[<list-of-eight-chars>]`.
-    '';
+    border = defaultNullOpts.mkBorder' {
+      pluginDefault = "single";
+      example = "rounded";
+      name = "all floating windows";
+    };
   };
+
   decorations = {
     statusline = {
       app_version = defaultNullOpts.mkBool false ''
         Set to true to be able use the `flutter_tools_decorations.app_version` in your statusline.
+
         This will show the current version of the flutter app from the `pubspec.yaml` file.
       '';
 
       device = defaultNullOpts.mkBool false ''
         Set to true to be able use the `flutter_tools_decorations.device` in your statusline.
+
         This will show the currently running device if an application was started with a specific
         device.
       '';
 
       project_config = defaultNullOpts.mkBool false ''
         Set to true to be able use the `flutter_tools_decorations.project_config` in your
-        statusline this will show the currently selected project configuration.
+        statusline.
+
+        This will show the currently selected project configuration.
       '';
     };
   };
 
   debugger = {
     enabled = defaultNullOpts.mkBool false ''
-      Integrate with nvim `dap` + install dart code debugger.
+      Enable `nvim-dap` integration.
```

Comment:
Why drop mention of the dart code debugger?

Reply:
I assumed it was the upstream plugin's wording. So the question isn't whether _we_ install it but whether _the plugin_ does 🤔

Reply:
Maybe. Looking at the upstream README there's several dependencies to consider... Happy for that to be tackled in other PRs though.

Reply:
I do see a telescope integration upstream, too. We could handle that in another PR.